### PR TITLE
1am9trash project proposal

### PR DESCRIPTION
**Pull request by 1am9trash**
---
A good PR should include the following parts:

- [x] A clear title
- [x] A meaningful message for PR, as well as its commits
- [x] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [x] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: project/1am9trash/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- test unit
+  - write in python
+  - both testing the correctness and speed
+- visualization unit
+  - write in python
+  - make complex polygon operations be intuitive
+
+## Api Description
+### C++
+- `vertex` class
```
Comment (yungyuc) [inline project/1am9trash/README.md:36]: Does it use real number?

File: project/1am9trash/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - define vertex
+- `polygon` class
+  - define polygon by `vertex`
+  - function checking polygon properties, such convex or not
+- `bool_operator_with_polygon` functions
+  - take polygons as arguments
+  - deal with and, or, xor, not operations
```
Comment (yungyuc) [inline project/1am9trash/README.md:43]: Look forward to the implementation of all of them.  It should be hard.

**Review by yungyuc — APPROVED** [review]
It's generally OK.  Leave a message when you want it to be merged.

File: project/1am9trash/README.md (hunk 1 above)
Comment (1am9trash) [reply project/1am9trash/README.md:36]: Yes, I may use double for high precision.

File: project/1am9trash/README.md (hunk 2 above)
Comment (1am9trash) [reply project/1am9trash/README.md:43]: I may try my best. Currently, I am stucked with tons of papers.

**Comment by 1am9trash** [comment]
It's ready to merge now. If there is any modification in the future, I may update proposal in my own repo.

**Pull request merged**